components/cam/src/physics/rrtmgp/radiation.F90

```diff
@@ -24,6 +24,20 @@ module radiation
    use mo_gas_optics_rrtmgp, only: ty_gas_optics_rrtmgp
    use mo_rte_kind, only: wp
 
+   ! Added for rte stuff
+   use mo_rte_kind,   only: wp
+   use mo_gas_optics, &
+                         only: ty_gas_optics
+   use mo_gas_concentrations, &
+                         only: ty_gas_concs
+   use mo_optical_props, only: ty_optical_props, &
+                               ty_optical_props_arry, ty_optical_props_1scl, ty_optical_props_2str, ty_optical_props_nstr
+   use mo_source_functions, &
+                         only: ty_source_func_lw
+   use mo_fluxes,        only: ty_fluxes
+   use mo_rte_lw,        only: base_rte_lw => rte_lw
```

Comment:
Do you need to remove some of these use statements to fully revert the code changes we were talking about?

Reply:
Good call. I don't think it will change anything, but removed and testing now.